src/content/whats-new/2024/04/whats-new-04-15-sessionreplayupdate

```diff
@@ -0,0 +1,24 @@
+---
+title: 'New Relic Session Replay Limited Preview Ends May 14, 2024'
+summary: 'Gain valuable insights and troubleshoot faster with a video-like playback of your user’s interactions'
+releaseDate: '2024-04-15'
```

Comment:
Confirm with @lchockalingam if this is supposed to go out today to meet the April 15th deadline, or if it needs to go out on May 15th with this date updated